### PR TITLE
Seed data and setup test

### PR DESCRIPTION
**Pull request by Nguyephi**
## Main purpose of this PR:

-  was to create seed data and use faker. I can insert 100 fake data,  but I wanted to refactor some things that will require me to use S3. 
    - [ ] Basically matching the name of the game to its poster.
    - [ ] Also need to figure out where to serve mp4 file to get the url needed.
- set up jest and enzyme to prepare for unit testing
- I also set up eslint (but I'm not sure if there are rules that we can or cannot use; I did not check)
- Other small commits were to accommodate for the new dependencies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,13 +5,16 @@
...
-    "react_dev": "webpack --mode development --watch"
+    "react_dev": "webpack --mode development --watch",
+    "seed": "node server/db/seed.js"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "dependencies": {
     "express": "^4.17.1",
+    "i": "^0.3.6",
```
Comment (kraig-droid) [inline package.json]: Huh, I didn't know npm-i is a package. Did you mean to install this or was it a typo?

**Review by kraig-droid — COMMENTED** [review]
The "main purpose" comment talks about using faker, but I don't see it anywhere. Did I miss it? Ditto for jest/enzyme setup.
For eslint, the package eslint-config-airbnb is the ticket. Read the docs or just install and watch for warnings on peer dependencies then install those.

File: package.json (hunk 1 above)
Comment (Nguyephi) [reply package.json]: Yea its not a package. I had another commit that removed that. Not sure why some commits pushed. Im looking into that right now.

**Comment by azdoolittle** [comment]
lgtm

**Review by priscilagonzalez — COMMENTED** [review]
Lgtm.

File: server/db/dummy-data.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    category: 'Super Smash Bros Ultimate',
+    game_poster_url:
+  }
+]
+
+let randomGameIdx = Math.floor(Math.random() * gameData.length);
+
+// this is to ensure that the total view is higher than watching now
+let randomTotalViews = Math.floor(Math.random() * (2000000 - 5000)) + 5000;
+let randomWatchingNow = Math.floor(Math.random() * (4500 - 2)) + 2;
```
Comment (kraig-droid) [inline server/db/dummy-data.js:58]: Nice...

**Comment by kraig-droid** [comment]
lgtm

**Pull request merged**